grpcui.bzl

```diff
@@ -0,0 +1,12 @@
+"""Bazel macro to run a binary copy of gRPC UI."""
+
+def grpcui(name, descriptor, args = []):
```

Comment:
What's this rule do?

Reply:
the idea was this rule would be a macro to invoke the binary from a user's bazel target, but we can exclude, because this would depend on the rest of the `rules_go` stuff.

Reply:
(or perhaps not? i suppose it could just invoke the binary and pass args, so yes, this would be a Bazel macro to invoke the binary from a target, rather than the command line. should you choose to keep it 😁)